### ISSUE TITLE
Core | Progress-circular issue: does not show in iOS 8

### ISSUE THREAD

**Issue by misterjunio** [OnsenUI/OnsenUI#1860]
__Environment__ 

```
[Core]
    onsenui 2.1.0

[Framework]
    none

[Framework binding]
    none

[Additional libraries]
    none

[Platform]
    Mobile - iOS 8.4.1

[Browser]
    Mobile - iOS Safari
```

__Encountered problem__
Progress-circular is not shown at all.

__How to reproduce__
Open the `progress` core example in iOS 8 Safari. The circular progress doesn't appear.

**Comment by asial-matagawa** [comment]
@misterjunio 
I have fixed this problem with #1867.

Closing this.

**Comment by NickPadilla** [comment]
I am seeing this issue in Fire Fox 52.0.2 

You can see it just using the onsen docs...

https://onsen.io/v2/docs/react/ProgressCircular.html

Is the reason why this is failing b/c of the element being an SVG so children is undefined?  If so, any work arounds?

**Comment by misterjunio** [comment]
@NickPadilla you're right, Firefox is not showing any progress-circular. @asial-matagawa do you think it may have something to do with your fix in #1867?

**Comment by asial-matagawa** [comment]
@misterjunio 
I fixed this problem between `2.1.0` and `2.2.0`, but #1840 broke `ons-progress-circular` again.

The current `ons-progress-circular` depends on SVG 2 features so it is not working on some browsers like iOS 8 and Firefox as this user explains:
https://github.com/OnsenUI/OnsenUI/pull/1840#pullrequestreview-27289015

**Comment by s-nel** [comment]
Can we revert the SVG 2 changes to fix issues in Firefox and iOS 8? They seem pretty superficial to me
